spec/models/cieszyn/job_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Cieszyn::Job, type: :model do
+  let(:job) { create(:job, name: 'lorem ipsum', price: 100) }
```

Comment:
You have factory with default values, so you don't need additional parameters. But you should fix your factory.